desktop/hardened-desktop/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hardened Desktop
+description: Overview of what Hardened Desktop is
+keywords: security, hardened desktop, enhanced container isolation, registry access management, admin controls, root access, administrators, docker desktop
+---
+>Note
+>
+>Hardened Desktop is available to Docker Business customers only.
+
+Hardened Desktop is a new security model for Docker Desktop. It's part of Docker's ongoing effort to increase Docker Desktop security without impacting the developer experience.
+
+The Hardened Desktop security model provides administrators with a simple and powerful way to increase the security of their containerised development and moves the ownership boundary for containers to the organization, meaning that any security controls administrators set cannot be altered by the user.
```

Comment:
then would condense this to:

"The Hardened Desktop security model moves the ownership boundary for containers to the organization, meaning that any security controls set by admins cannot be altered by the user of Docker Desktop."